### DISCUSSION TITLE
copy from clipboard in command mode

### DISCUSSION THREAD

**Discussion by PatrickStahle** [helix-editor/helix#9105] [Q&A]
Hi,

I seem unable to copy from clipboard when in command mode ':'. For instance trying to do an open ":open " and paste the file path/name in to open a file. It seems ignore the paste and does nothing. Am I missing something?

Thanks,
Patrick

**Comment by kirawi** [comment]
Are you able to paste outside of Helix? If not, then see https://github.com/helix-editor/helix/wiki/Troubleshooting#copypaste-fromto-system-clipboard-not-working

**Reply by PatrickStahle** [reply]
I am able to paste from clipboard everywhere including in helix in insert mode. Just not in command mode like I can in vim. Try my example and see if it works for you (:open /paste something/)?

**Reply by TornaxO7** [reply]
how are you executing the paste command in helix command mode? Do you use `Ctrl` + `Shift` + `v`?

**Reply by PatrickStahle** [reply]
To answer your question first, I was using my mouse button. I now started to play with it more and discovered that under tmux both ctrl+shift+v and middle mouse / shift middle mouse are working fine. With a new terminal and no tmux ctrl+shift+v and shift middle mouse is working, but not plain middle mouse button. I really would of thought it would of been the opposite.... I am on Fedora 39 and using gnome terminal. I am curious to see what happens under gnu screen. Looking like this might be just some weirdness with my environment....?

**Reply by PatrickStahle** [reply]
GNU Screen in regards to clipboard acts the same as it does without tmux, so ctrl+shift+v works and shift+mouse button works, but not just mouse button. Though with gnu screen I have issues with colors being messed up. I have tried:  TERM  = "screen.xterm-256color", "tmux-256color", "xterm" with same result. Though I guess that is a different  topic.

**Reply by TornaxO7** [reply]
Aye, seems to be unrelated to helix.

**Reply by kirawi** [reply]
You can use `C-r` in the prompt as an alternative. I suspefct this may have something to do with bracketed paste but I'll need to test later.
